package.json

```diff
@@ -54,7 +50,10 @@
     "@babel/preset-env": "^7.15.0",
     "@babel/preset-typescript": "^7.14.5",
     "@emotion/babel-plugin": "^11.3.0",
-    "@emotion/react": "^11.1.2",
+    "@emotion/react": "11.0.0",
+    "@guardian/eslint-config": "^0.7.0",
+    "@guardian/eslint-config-typescript": "^0.7.0",
```

Comment:
Do we need to install both eslint-config and eslint-config-typescript in Source? I expected @guardian/eslint-config-typescript to extend @guardian/eslint-config and thus you'd only need @guardian/eslint-config-typescript, but maybe I got that wrong 😅